doc/reference/devices_nic.md

```diff
@@ -278,6 +329,17 @@ Key                                   | Type    | Default           | Managed |
 `security.acls.default.ingress.logged`| bool    | `false`           | no      | Whether to log ingress traffic that doesn't match any ACL rule
 `vlan`                                | integer | -                 | no      | The VLAN ID to use when nesting (see also `nested`)
 
+#### Configuration examples
+
+An `ovn` network device must be added using a managed network.
+To do so:
+
+    lxc network set <parent_network> ipv4.dhcp.ranges=<IP_range> ipv4.ovn.ranges=<IP_range>
```

Comment:
the other NIC types dont cover setting up the managed network so it seems odd that we do for OVN NIC only.